pkg/argsbuilder/submit_tfjob.go

```diff
@@ -130,6 +130,8 @@ func (s *SubmitTFJobArgsBuilder) AddCommandFlags(command *cobra.Command) {
 	command.Flags().StringVar(&s.args.PSMemory, "ps-memory", "", "the memory resource to use for the parameter servers, like 1Gi.")
 	command.Flags().StringVar(&s.args.PSMemoryLimit, "ps-memory-limit", "", "the memory resource limit to use for the parameter servers, like 1Gi.")
 
+	command.Flags().StringVar(&s.args.SuccessPolicy, "success-policy", "", "Specifies the policy to mark the TFJob as succeeded. Available options are \"\" and AllWorkers. Default to \"\".")
```

Comment:
Available Value: AllWorkers and Chief？

Reply:
Yes, I think need clarify what default value indicates.